maps/tether/submaps/offmap/talon1.dmm

```diff
@@ -216,9 +219,76 @@
 /obj/machinery/chemical_dispenser/full,
 /turf/simulated/floor/tiled/eris/white/bluecorner,
 /area/talon/deckone/medical)
+"ax" = (
+/obj/machinery/camera/network/talon,
+/obj/structure/closet/wardrobe/black{
+	starts_with = list(/obj/item/clothing/head/helmet/combat = 4, /obj/item/clothing/mask/gas/commando = 4, /obj/item/clothing/mask/balaclava = 4, /obj/item/clothing/under/syndicate/combat = 4, /obj/item/clothing/accessory/storage/black_vest = 4, /obj/item/clothing/accessory/storage/black_drop_pouches = 4, /obj/item/clothing/gloves/combat = 4, /obj/item/clothing/suit/armor/combat = 4, /obj/item/clothing/shoes/boots/combat = 4)
+	},
+/turf/simulated/floor/tiled/eris/white/danger,
+/area/talon/deckone/armory)
+"ay" = (
+/obj/structure/catwalk,
+/obj/structure/cable/green{
+	d1 = 1;
+	d2 = 2;
+	icon_state = "1-2"
+	},
+/obj/machinery/light/small{
+	dir = 4;
+	pixel_y = 0
+	},
+/turf/simulated/floor/plating/eris/under,
+/area/talon/maintenance/deckone_port)
+"az" = (
+/obj/structure/catwalk,
+/obj/structure/cable/green{
+	d1 = 1;
+	d2 = 2;
+	icon_state = "1-2"
+	},
+/obj/machinery/light/small{
+	dir = 8;
+	pixel_x = 0
+	},
+/turf/simulated/floor/plating/eris/under,
+/area/talon/maintenance/deckone_starboard)
+"aA" = (
+/obj/structure/catwalk,
+/obj/structure/cable/green{
+	d1 = 1;
+	d2 = 4;
+	icon_state = "1-4"
+	},
+/obj/structure/cable/green{
+	d1 = 4;
+	d2 = 8;
+	icon_state = "4-8"
+	},
+/obj/machinery/light/small,
+/turf/simulated/floor/plating/eris/under,
+/area/talon/maintenance/deckone_port)
+"aB" = (
+/obj/structure/catwalk,
+/obj/structure/cable/green{
+	icon_state = "1-8"
+	},
+/obj/structure/cable/green{
+	d1 = 4;
+	d2 = 8;
+	icon_state = "4-8"
+	},
+/obj/machinery/light/small,
+/turf/simulated/floor/plating/eris/under,
+/area/talon/maintenance/deckone_starboard)
 "aC" = (
 /turf/simulated/wall/rshull,
 /area/talon/maintenance/deckone_port)
+"aD" = (
+/obj/structure/closet/wardrobe/black{
+	starts_with = list(/obj/item/clothing/head/helmet/space/syndicate/black = 4, /obj/item/clothing/suit/space/syndicate/black = 4, /obj/item/clothing/shoes/magboots = 4)
```

Comment:
Here too, four sets of the armored syndicate spacesuits instead of the current one and two normal unarmored talon crew suits.

Reply:
> 
> 
> I don't think this counts as a "minor edit", this is giving the talon four full sets of combat armor, while currently they get a single helmet, armor suit, and jumpsuit. Those gloves are heavily armored and insulated, while the boots are noslips. Why does the talon need to be armored up like they're nuke ops?

Wasn't sure how armored up they should be. I take it that one set is enough then?

Had no idea the gloves and boots were that special. Yikes! I'll downgrade those to somehting a bit more reasonable.

The locker thing is good to know too. I'll undo that change.

> 
> 
> Here too, four sets of the armored syndicate spacesuits instead of the current one and two normal unarmored talon crew suits, plus a specialized suit that's in each crew locker.

Whoops, forgot about the special suits. Retracting that too!